### ISSUE TITLE
3.0 - Download stats re-count tool is counting refunds

### ISSUE THREAD

**Issue by ashleyfae** [easydigitaldownloads/easy-digital-downloads#8659]
### Expected behavior

Download stats should be of completed _sales_. I wouldn't see any refund information.

### Actual behavior

Refund records are skewing the results.

### Steps to reproduce the behavior
1) For easier testing, create a brand new product.
2) Purchase the product once.
3) Then refund that purchase.
4) Now the sale & earnings counts. For me:
![Screenshot from 2021-05-26 13-44-20](https://user-images.githubusercontent.com/6324272/119661740-812b4d00-be28-11eb-94f7-eb70ea7feb74.png)
5) Then go to Downloads > Tools and in "Recount Stats", re-count for the download you just created.
6) Re-check the numbers.
![Screenshot from 2021-05-26 13-45-09](https://user-images.githubusercontent.com/6324272/119663226-12e78a00-be2a-11eb-9a20-8d665eafe4fa.png)

What's actually happening here is that the recount tool does a query like this:

```php
$order_items = edd_get_order_items( array(
    'status__in' => array( 'complete', 'revoked' ),
    'product_id' => $your_product_id
) );
```

Well the original order record is `refunded`, so that won't get picked up at all. Instead it picks up the _refund_ record (which is order_type = refund; status = complete). It adds that amount to the earnings, which is actually a negative number (`-10.93` for me). But on display, the earnings isn't allowed to be below `0`, so it just displays `0` instead.

tl;dr: This whole thing is weird.

Copying from Slack discussion: I like @spencerfinnell's line of thinking which is that:

1. Sales number should always be number of orders. Maybe orders of any status that would have been complete at one time. (`edd_get_gross_order_statuses()`)
2. `Earnings` column should be re-named to `Revenue`.
3. Earnings is the same as 1, but for earnings instead of sales count.

With that in mind... Perhaps the easiest solution here is to add a new argument to `edd_get_order_items()` that would allow you to pass in `'order_type' => 'sale'`. Then we could make sure we only actually pick up sales and not refunds.

### Information (if a specific version is affected):
PHP Version:

EDD Version (or branch): release/3.0

WordPress Version:

Any other relevant information:

**Comment by ashleyfae** [comment]
More from Slack:

> If we choose for it to include refunds, it needs to at least be gross or have both, if wechoose to not have refunds in there then it needs to have a prefix of net

tl;dr: We should rename the column to `Gross Revenue`
